apps/dashboard/src/main/java/com/akto/action/ApiCollectionsAction.java

```diff
@@ -110,6 +123,73 @@ public String deleteMultipleCollections() {
         return SUCCESS.toUpperCase();
     }
 
+    // required for icons and total sensitive endpoints in collections
+    public String fetchSensitiveInfoInCollections(){
+        this.sensitiveInfoInApiCollections = SingleTypeInfoDao.instance.getSensitiveInfoForCollections() ;
+        return Action.SUCCESS.toUpperCase();
+    }
+
+    // required to measure the count of total tested endpoints per collection.
+    public String fetchCoverageInfoInCollections(){
+        this.testedEndpointsMaps = ApiInfoDao.instance.getCoverageCount();
+        return Action.SUCCESS.toUpperCase();
+    }
+
+    // required to measure the count of total issues per collection.
+    public String fetchSeverityInfoInCollections(){
+        this.severityInfo = TestingRunIssuesDao.instance.getSeveritiesMapForCollections();
+        return Action.SUCCESS.toUpperCase();
+    }
+
+    public String fetchLastSeenInfoInCollections(){
+        this.lastTrafficSeenMap = ApiInfoDao.instance.getLastTrafficSeen();
+        return Action.SUCCESS.toUpperCase();
+    }
+
+    public String riskScoreInfo(){
+        int criticalCount = 0 ;
+        Map<Integer, Double> riskScoreMap = new HashMap<>();
+        List<Bson> pipeline = ApiInfoDao.instance.getFiltersForRiskScore();
+
+        MongoCursor<BasicDBObject> apiCursor = ApiInfoDao.instance.getMCollection().aggregate(pipeline, BasicDBObject.class).cursor();
+        while(apiCursor.hasNext()){
+            try {
+                BasicDBObject basicDBObject = apiCursor.next();
+                double riskScore = basicDBObject.getDouble("riskScore");
+                BasicDBObject bd = (BasicDBObject) basicDBObject.get("_id");
+                Integer collectionId = bd.getInt("apiCollectionId");
+
+                // store count of total critical endpoints present in ApiInfo
+                if(riskScore >= 4){
+                    criticalCount++;
+                }
+
+                // as for collections, risk score is max of apis in it, here we take max for collection id
+                if(riskScoreMap.isEmpty() || !riskScoreMap.containsKey(collectionId)){
+                    riskScoreMap.put(collectionId, riskScore);
+                }else{
+                    double prev = riskScoreMap.get(collectionId);
+                    riskScore = Math.max(riskScore, prev);
+                    riskScoreMap.put(collectionId, riskScore);
+                }
+
+            } catch (Exception e) {
+                // TODO: handle exception
+                e.printStackTrace();
+            }
+        }
+
+        this.criticalEndpointsCount = criticalCount;
+        this.riskScoreOfCollectionsMap = riskScoreMap;
+        return Action.SUCCESS.toUpperCase();
+    }
+    
+    public String fetchTimersInfo(){
+        CronTimers timer = AccountSettingsDao.instance.getTimersInfo();
```

Comment:
i can see this is still not fixed. Can you please rename CronTimers and getTimersInfo, really confusing

Reply:
done